### ISSUE TITLE
EP-DNN: A Deep Neural Network-Based Global Enhancer Prediction Algorithm

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#154]
http://doi.org/10.1038/srep38433

> We present EP-DNN, a protocol for predicting enhancers based on chromatin features, in different cell types. Specifically, we use a deep neural network (DNN)-based architecture to extract enhancer signatures in a representative human embryonic stem cell type (H1) and a differentiated lung cell type (IMR90). We train EP-DNN using p300 binding sites, as enhancers, and TSS and random non-DHS sites, as non-enhancers. We perform same-cell and cross-cell predictions to quantify the validation rate and compare against two state-of-the-art methods, DEEP-ENCODE and RFECS. We find that EP-DNN has superior accuracy with a validation rate of 91.6%, relative to 85.3% for DEEP-ENCODE and 85.5% for RFECS, for a given number of enhancer predictions and also scales better for a larger number of enhancer predictions. Moreover, our H1 → IMR90 predictions turn out to be more accurate than IMR90 → IMR90, potentially because H1 exhibits a richer signature set and our EP-DNN model is expressive enough to extract these subtleties. Our work shows how to leverage the full expressivity of deep learning models, using multiple hidden layers, while avoiding overfitting on the training data. We also lay the foundation for exploration of cross-cell enhancer predictions, potentially reducing the need for expensive experimentation.

Figure 4 shows the design: "EP-DNN is a fully connected DNN with an 80-600-500-400-1 architecture and softplus activation functions...  It takes 4 histone modifications (20 features in each mod, with ten 100 bp bins on each side of a location) as input and has a single real valued output which is put through a threshold to determine the classification of a potential enhancer location."
